### PR TITLE
Add Kfserving SDK notebook example

### PR DESCRIPTION
**Pull request by PatrickXYS**
*Issue #24 , if available:*

*Description of changes:*
Add one Kfserving SDK notebook example.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by PatrickXYS** [comment]
@Jeffwan Can you take a look?

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+   "outputs": [],
+   "source": [
+    "KFServing.get('flower-sample', namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Add Canary to InferenceService"
```
Comment (Jeffwan) [inline notebooks/06_Serving/06_02_kfserving_sdk.ipynb:150]: Before this, we need to show how to invoke this endpoint.

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+    "\n",
+    "KFServing.rollout_canary('flower-sample', canary=canary_endpoint_spec, percent=10,\n",
+    "                         namespace=namespace, watch=True, timeout_seconds=120)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Rollout more traffic to canary of the InferenceService"
```
Comment (Jeffwan) [inline notebooks/06_Serving/06_02_kfserving_sdk.ipynb:182]: after this step, user can query service again, for example run 10 times, they will see ~5 from original endpoint, and another 5 from canary.

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb [hunk 3]
```diff
@@ -0,0 +1,256 @@
+{
```
Comment (Jeffwan) [inline notebooks/06_Serving/06_02_kfserving_sdk.ipynb:1]: Can we have a new section? 
1. Using YAML to deploy inferenceService
2. Using SDK to deploy the inferenceService

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+ "cells": [
```
Comment (Jeffwan) [inline notebooks/06_Serving/06_02_kfserving_sdk.ipynb:2]: Can you add a section to use S3 as well? Lots of user ask how to load model from S3.

Comment (Jeffwan) [reply notebooks/06_Serving/06_02_kfserving_sdk.ipynb:2]: If you can not easily figure this out, let me know. I can find previous example for you.

Comment (PatrickXYS) [reply notebooks/06_Serving/06_02_kfserving_sdk.ipynb:2]: Yeah we could, that will be proposed in the next PR.

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb (hunk 3 above)
Comment (PatrickXYS) [reply notebooks/06_Serving/06_02_kfserving_sdk.ipynb:1]: It makes sense.

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb (hunk 1 above)
Comment (PatrickXYS) [reply notebooks/06_Serving/06_02_kfserving_sdk.ipynb:150]: I can tell users how to invoke endpoint from users' local environment, but I can't find a way from SDK side.

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb (hunk 2 above)
Comment (PatrickXYS) [reply notebooks/06_Serving/06_02_kfserving_sdk.ipynb:231]: Addressed in the new commit.

File: notebooks/06_Serving/06_02_kfserving_sdk.ipynb (hunk 1 above)
Comment (Jeffwan) [reply notebooks/06_Serving/06_02_kfserving_sdk.ipynb:195]: It's fine. Curl HTTP request will be helpful @PatrickXYS

**Comment by PatrickXYS** [comment]
Combine this PR with #64 .

**Pull request closed without merging**